scripts/release/create_changelog.py

```diff
@@ -72,7 +72,6 @@
     with open(changelog_path, "w") as f:
         # Add frontmatter based on args
         f.write("---\n")
-        f.write(f"title: {title}\n")
```

Comment:
I'm happy with the PR as it is, but wondering - would it make sense to render title as a bullet point in the release note body? It can be used as a starting point for a release note.

Reply:
Yeah, I think this is a good idea!

Reply:
Done!